CONTRIBUTING.md

```diff
@@ -32,6 +32,23 @@ We welcome contributions for new generators to extend the functionality of the O
 
 11. **Address Feedback**: Be responsive to feedback from the maintainers. Make any necessary changes and update your pull request as needed.
 
+### Integration Testing
+
+To verify that generated code compiles correctly, the project includes integration tests, for example, for c#:
+
+```bash
+# Test the C# generator output
+make test-csharp
+```
+
+This will:
+1. Build the CLI
+2. Generate a C# client
+3. Compile the C# code in a Docker container
+4. Validate that the code compiles correctly
+
+Consider adding more integration tests for new generators.
```

Comment:
I could just include adding integration tests into the contribution notes above, but wanted to hear feedback from team before including an additional step in contribution guidelines.

Reply:
We talked about this at KubeCon, but you beat me to the implementation!

Reply:
![image](https://github.com/user-attachments/assets/8ef01257-6239-468e-8f6a-930d939f1374)